src/main/java/org/opensearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -40,6 +40,7 @@
 import com.fasterxml.jackson.annotation.JsonAnyGetter;
 import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
```

Comment:
How is the changes applied to `ConfigV6` tested?

Reply:
Not able to test changes applied to `ConfigV6`. Keeping the changes to maintain consistency with `ConfigV7`.

Reply:
please create 6x domain to test those changes.

Reply:
As discussed, this is mainly for 6.x to 7.x migration and will not impact 6.x directly. But lets test migration case as discussed offline.